templates/documents/repositories.html

```diff
@@ -0,0 +1,47 @@
+{# Generic template to render a list of objects with a 'new' button #}
+{% extends 'web/app/app_base.html' %}
+{% block app %}
+    {% load i18n %}
+    <div class="app-card" x-data>
+        <div class="grid grid-cols-6">
+            <div class="col-span-5">
+                <div class="flex">
+                    <h1 id="{{ title|slugify }}" class="{{ title_class|default:"pg-title" }} hover:cursor-pointer">
+                        <a href="#{{ title|slugify }}">{{ title }}</a>
+                    </h1>
+                </div>
+            </div>
+        </div>
+
+        <div role="tablist" class="tabs tabs-bordered">
+            <input
+                type="radio"
+                name="repository"
+                role="tab"
+                class="tab"
+                aria-label="Files ({{ files_count }})"
+                @click="window.location.href = '{% url 'documents:repositories' request.team.slug 'files' %}'"
```

Comment:
Since you're loading a new page when changing tabs I think it makes more sense to make this styled links instead of an actual tab list.

Reply:
I'm curious what your reasoning behind this is?

Reply:
That makes sense. See https://github.com/dimagi/open-chat-studio/pull/1259/commits/39a343c4ab1629ad66674a4ce75ead4114ca823e

Screenshot
![image](https://github.com/user-attachments/assets/30b12614-2727-48ef-8a70-3ac8b08d8215)